i18n/translations/en.ts

```diff
@@ -260,6 +260,7 @@ const en = {
   group_join_error: "An error occurred",
   group_join_invite_invalid: "This invite is no longer valid",
   group_finished_polling: "This is taking longer than expected",
```

Comment:
Since this correlates with `finishedPollingUnsuccessfully`, we should rename the string key to contain that `_unsuccessfully` condition.

Reply:
or `without_being_able_to_join` or something to indicate that this string comes along with some undesirable condition